### DISCUSSION TITLE
would you guys use a custom zoom for sprite editing/making? or hd png export?

### DISCUSSION THREAD

**Discussion by SarahKomik** [ScratchAddons/ScratchAddons#7014] [Ideas]
i usually make scratch sprites for my yt videos, yet I have to use a restricted zoom to screenshot things, it would be nice if there were an addon for custom zoom or even export sprites as pngs!!!

**Comment by BroJac5246** [comment]
You can right-click a costume to export it as an SVG. From there you can convert it to a PNG.

Edit: that's for vector; bitmap might already export as a PNG.
